torchtext/experimental/transforms/transforms.py

```diff
@@ -0,0 +1,86 @@
+from typing import List
+import torch
+from torchtext.data.functional import load_sp_model
+
+
+class SplitTokenizer(torch.nn.Module):
```

Comment:
I think this should be supported via a RegexTokenizer. It's also not covered by the tests. Even the functionality might be obvious, at the very minimum a test prevents syntax errors.

Reply:
Yes. We can re-write the re package with JIT support. It's on the plan.

Reply:
Let's do that first then before we introduce an abstraction we plan to immediately after remove. Or should we have a SplitTokenizer in the long term as well?

Reply:
I think we can have different tokenizer transforms around. For example, an sentencepiece transform accept a sp model in the constructor and convert string into tokens. Another one could be convert string into index. With those two, it covers some usage cases in pytext.